inc/impact.class.php

```diff
@@ -156,24 +156,30 @@ public static function displayTabContentForItem(
          $linked_items = $item->getLinkedItems();
 
          // Search for a valid linked item of this ITILObject
-         $found = false;
+         $items_data = [];
          foreach ($linked_items as $itemtype => $linked_item_ids) {
             $class = $itemtype;
             if (self::isEnabled($class)) {
                $item = new $class;
                foreach ($linked_item_ids as $linked_item_id) {
-                  $found = $item->getFromDB($linked_item_id);
-                  break 2;
+                  if ($item->getFromDB($linked_item_id)) {
+                     $items_data[] = [
+                        'itemtype' => $itemtype,
+                        'items_id' => $linked_item_id,
+                        'name'     => $item->getNameID(),
+                     ];
+                  }
+                  continue;
```

Comment:
`continue` statement doesn't seem needed since this is already the end of the loop.

Reply:
I reorganized code.